.github/workflows/changelog_checker.yml

```diff
@@ -0,0 +1,19 @@
+name: "Changelog checker"
+on:
+  pull_request:
+    # The specific activity types are listed here to include "labeled" and "unlabeled"
+    # (which are not included by default for the "pull_request" trigger).
+    # This is needed to allow skipping enforcement of the changelog in PRs with specific labels,
+    # as defined in the (optional) "skipLabels" property.
```

Comment:
Why can't the "skipLabels" property be used without including `labeled` and `unlabeled` in types?

Reply:
Right, so this text is actually copied from a template so I hope I'm right. The point I'm pretty sure is, that the test will not be triggered when you **change** the label, unless you make it listen for those event. The regular CI Github Actions are only run on PR creation, open/close and new commits. So, of you were missing the change log entry and added in a new commit, the test would be re-run and pass. But if you were missing it and decided that the PR shouldn't have one and add the label, the tests will not be re-run and the test will remain failed... Unless you listen for the label changes events.